app/javascript/src/js/mdPreviewShare.test.js

```diff
@@ -0,0 +1,49 @@
+import { generateLink, encodeContent, decodeContent } from './mdPreviewShare';
+
+describe('encodeContent', () => {
+  it('content is correctly encoded to base64', () => {
+    const buffer = Buffer.from('This is a test string!', 'utf-8');
+    expect(encodeContent('This is a test string!')).toBe(buffer.toString('base64'));
+  });
+  it('content including numbers is correctly encoded to base64', () => {
```

Comment:
Nitpick, can we separate each test with a new line please? just to make it easier a little easier to read.

Reply:
Of course! Still kind of getting used to spacing conventions :)